### PR TITLE
Will add the exception message to the report. Will use the suite name fo...

### PR DESCRIPTION
**Pull request by ghost**
I use PSR-2 for formatting. Sorry I should do it in a separate commit.

The title of the report will have the name of the suite.

In case of exception will include the message to the report.

### CODE REVIEW

File: src/chartinger/Behat/TwigReportExtension/views/default.twig [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% macro result_panel_class(result) %}
+EventListener.php{% macro result_panel_class(result) %}
```
Comment (mithikos) [inline src/chartinger/Behat/TwigReportExtension/views/default.twig]: C/P error here... can me removed...

File: src/chartinger/Behat/TwigReportExtension/views/default.twig [hunk 2]
```diff
@@ -97,7 +97,15 @@
                   {% for step in feature.background.steps %}
                      <li class="list-group-item {% if step.result == 20 %}list-group-item-warning{% elseif step.result == 10%}disabled{% elseif step.result == 99 %}list-group-item-danger{% else %}list-group-item-success{% endif %}">
                      {{ default.result_glyph(step.result) }}
-                     {{ step.text }}
+                     {{ step.text }}
```
Comment (mithikos) [inline src/chartinger/Behat/TwigReportExtension/views/default.twig]: I haven't work before with twig templates. I don't know if there is more neat way to do this.

**Comment by mithikos** [comment]
Any comment will be nice :P

**Comment by chartinger** [comment]
Sorry it took so long. Nice addition, but the suite name does not seem to work, it returns the event name which is not all that useful.

Good thing to change to PSR-2 formatting, i will do a reformatting of the other files after this pull request.

Maybe add an exception message to one of the example features to see it in action?

**Comment by chartinger** [comment]
Update: it only does not work if there is no output directory (and file per suite)

**Comment by mithikos** [comment]
Thank you for your comments. I will investigate.

**Pull request closed without merging**